### ISSUE TITLE
Image pulled successfully, but not visible in UI, not repository created

### ISSUE THREAD

**Issue by osaffer** [goharbor/harbor#13207]
If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**
I expect the creation of the repository after pushing the image

**Steps to reproduce the problem:**
Please provide the steps to reproduce this problem.

**Versions:**
Please specify the versions of following systems.

- harbor version: CHART Version 7.4.0     APP Version   	2.0.2      
- docker version
Client: Docker Engine - Community
 Version:           19.03.12
 API version:       1.39

- docker-compose version 1.17.1
kubectl version --short
Client Version: v1.19.0
Server Version: v1.19.0

Installation done via Helm

docker push my-harbor-host/library/debian:4.0
The push refers to repository [my-harbor-host/library/debian]
4ef54afed780: Pushed 
4.0: digest: sha256:5658c83d366defa40f1e43cd9dbee4c8af2d86c31878f41296978b4cea4344bc size: 529

wget https://register:register@my-harbor-host/v2/_catalog
cat _catalog 
{"repositories":["library/debian"]}

in harbor-core  pod: 
/server/middleware/security/unauthorized.go:29][requestID="1cd21632-c4bc-4f0f-99c1-46fbf64c23e2"]: an unauthorized security context generated for request POST /service/notifications

in harbor-registry pod:
retryingsink: error writing events: httpSink{http://harbor-core/service/notifications}: response status 404 Not Found unaccepted, retrying" 
time="2020-10-01T14:55:45.462209951Z" level=warning msg="httpSink{http://harbor-core/service/notifications} encountered too many errors, backing off" 

in harbor-notary:

 Starting Harbor Notary Server **
{"level":"info","msg":"Version: 0.6.1, Git commit: d6e1431f","time":"2020-10-01T14:06:41Z"}
{"level":"info","msg":"Using remote signing service","time":"2020-10-01T14:06:41Z"}
{"level":"info","msg":"Using postgres backend","time":"2020-10-01T14:06:41Z"}
{"level":"info","msg":"Starting Server","time":"2020-10-01T14:06:41Z"}
{"level":"info","msg":"Starting on :4443","time":"2020-10-01T14:06:41Z"}
2020/10/01 14:06:42 grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 172.26.196.72:7899: connect: connection refused"; Reconnecting to {harbor-notary-signer:7899 <nil>}

Thanks a lot for your support.

**Comment by osaffer** [comment]
kubectl exec -it pod/harbor-core-65b4d8498d-v2lsl -n harbor -- sh
$ bash
I have no name!@harbor-core-65b4d8498d-v2lsl:/opt/bitnami/harbor-core$ curl http://harbor-core/service/notifications
<!DOCTYPE HTML>
<html>
<head>
<meta charset="utf-8">
<title>404 - Page Not Found</title>
<style>

**Comment by steven-zou** [comment]
Which chart repo are you using？ Bitnami or official?

**Comment by osaffer** [comment]
Hi,

Bitnami repo

Regards

**Comment by osaffer** [comment]
But now, I have just installed the harbor from Harbor repo, and it helped me to get rid of this error : " level=warning msg="httpSink{http://harbor-core/service/notifications} encountered too many errors, backing off"

I have deployed harbor from harbor repo exactly from the same way.

Other problem I faced with bitnami, was, when I modified proxy values to set our proxy server, the variable HTTP_PROXY ,HTTPS_PROXY and NO_PROXY were not set, so I was obliged to modified in CM, or Deployment of some elements as trivy, clair . 
I tried to set this variable with quotes, double quotes, without quotes ... but it did not work.
With Harbor from Harbor repo, no problem.
